OxfordBodleian/Ullendorff/BDLeu6.xml

```diff
@@ -130,21 +173,96 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
+                                    form), and elements of the numerals and of the punctuation</seg>
+                                <list type="abbreviations">
+                                    <item>
+                                        <abbr>ሰላ</abbr> for <expan>ሰላም፡</expan> (<locus from="1v" to="6r"/>)
+                                    </item>
+                                    <item>
+                                        <abbr>ሩፋ</abbr> or <abbr>ሩፋኤ</abbr> for <expan>ሩፋኤል፡</expan> (<locus from="3r" to="6r"/>)
+                                    </item>
+                                </list>
+                       </handNote>
                         </handDesc>
                         
                        <!-- <decoDesc>
-                         
                         </decoDesc>-->
                         
+                        <additions>
+                            <list>
+                                <item xml:id="a1">
+                                    <desc type="GuestText"><title ref="LIT2849RepCh131"/>
+                                        The note contains several corrections and additions in the margins, referenced in the body of the text by means of 
+                                        <foreign xml:lang="gez">tamallas</foreign>-signs (<locus target="#1v #6r"/>). 
+                                    </desc>
+                                    <locus from="1v" to="7v"/>
+                                    <q xml:lang="gez">
+                                        <seg part="I">
+                                            <hi rend="rubric">ሰላም፡ ለዝክረ፡ ስምከ፡</hi> ተጽውዖ፡ ዘቀደመ፡ በአፈ፡ አምላክ፡ አዳም፡ እንበለ፡ ይሰመይ፡ ስመ፡ 
+                                            <hi rend="rubric">ሩፋኤል፡</hi> ምልዓኒ፡ ወወስከኒ፡ ዳግመ፡ መንፈስ፡ ቅዱሳዌ፡ አእምሮ፡ እንተ፡ ይፈሪ፡ ሰላመ፡
+                                        </seg>
+                                    <seg part="F">
+                                        <hi rend="rubric">ሰላም፡</hi> ዕጣነ፡ ጸሎትየ፡ አዕርግ፡ እግዚኦ፡ ኀበ፡ አምላክ፡ ልዑል። በማዕጠንትከ፡ ዘፍያል፡ <hi rend="rubric">ሩፋኤል<supplied reason="omitted">፡</supplied></hi> 
+                                        ሊቅ<supplied reason="omitted">፡</supplied> ካህነ፡ ወንጌል። ወባልሔ፡ ኩነኒ፡ እምሕጕል። 
+                                        <hi rend="rubric">ኦአምላከ፡ ሩፋኤል፡ አድኀነኒ፡ እምነ፡ ሐጕል።</hi>
+                                    </seg>
+                                   </q>
+                                </item>
+                                <item xml:id="a2">
+                                    <desc type="">
+                                        The note is written in Arabic on the inner side of the back board.
+                                    </desc>
+                                    <q xml:lang="ar">
+                                        <!-- Please help with this text -->
```

Comment:
Not easy! I also didn't manage to read everything. Here are suggestions for some parts of (if you agree with my interpretations, this would point to the note being a summary of the manuscript's content):
مديح... لمار روفايل الملاك
سيرة الانبا ... الرهبنة
الانطونية في الحبشة وجدول...
تسلسل الرهبان...
Praise...of the angel Mār Rūfāʾīl
Life of ʾAnbā ... (I would like the following word to be اراجاوي, but this doesn't seem to be the case...) 
Anthonian monasticism in Ethiopia and schedule...
...of the sequence of monks....

Reply:
Thank you Thea! It's exactly what you suggest: a short summery of the book content. After سيرة الانبا I read اراݣاۋ o اراݣاڨى, i.e. Aragawi. The other words are also difficult to me